tests/features/content_editing_admin.feature

```diff
@@ -48,3 +53,32 @@ Feature: Content editing as administrator
       | This is not the right way                                             | <div class=\"classname?&*\">Applied invalid css class</div>           | classname                                                               |
       | This is not the right way                                             | <div id=\"2invalidid\">A container with an invalid HTML ID</div>      | invalidid                                                               |
       | This is not the right way                                             | <div id=\"invalidid.\">A container with an invalid HTML ID</div>      | invalidid                                                               |
+
+
+  Scenario Outline: Check admin UI always shows english
+    When I go to "admin/config/regional/translate/translate"
+    And I fill in "String contains" with "Body"
+    And I press "Filter"
+    And I click "edit" in the "field_ne_body:page:label" row
+    And I fill in "French" with "Corps du texte"
+    And I fill in "Italian" with "Corpo del testo"
+    And I press "Save translations"
+    Then I should see the following success messages:
```

Comment:
To check that a single message, we should use the step:
'Then I should see the success message'